internal/ingress/controller/template/template.go

```diff
@@ -268,6 +268,8 @@ func buildLuaSharedDictionaries(c interface{}, s interface{}, disableLuaRestyWAF
 		}
 	}
 
+	out = append(out, "lua_shared_dict certificate_servers 5M")
```

Comment:
this should be defined in https://github.com/kubernetes/ingress-nginx/blob/4847bb02f05ab323afb578600e83e4879a9791c7/internal/ingress/controller/template/configmap.go#L67

Reply:
done